mods/d2k/rules/defaults.yaml

```diff
@@ -336,8 +336,13 @@
 		VisibilityType: CenterPosition
 	Targetable:
 		TargetTypes: Ground, C4, Structure
+	BodyOrientation:
+		CameraPitch: 255
```

Comment:
The camera pitch affects the custom `TargetableOffsets`, which would be far more difficult to set with default pitch (512 would be far less than half a cell along the vertical axis).